src/Executables/ReduceCceWorldtube/ReduceCceWorldtube.cpp

```diff
@@ -291,7 +291,9 @@ int main(int argc, char** argv) {
       "input_file", boost::program_options::value<std::string>()->required(),
       "name of old CCE data file")(
       "output_file", boost::program_options::value<std::string>()->required(),
-      "output filename")(
+      "output filename")("fix_spec_normalization",
```

Comment:
please insert a line brake between `(` and `"fix_spec` to be consistent with the formatting above and below